send-transaction-service/src/send_transaction_service.rs

```diff
@@ -186,60 +187,60 @@ where
 
 /// Metrics of the send-transaction-service.
 #[derive(Default)]
-struct SendTransactionServiceStats {
+pub struct SendTransactionServiceStats {
```

Comment:
It became pub because of public trait `TransactionClient`

Reply:
`TransactionClient` will be used outside of this crate. The user will instantiate an implementation of this trait on his/her side. There are two implementations: one using ConnectionCache and the other one using tpu-client-next.